assets/src/data/model/datetime/query.js

```diff
@@ -35,6 +39,19 @@ export const queryDataTypes = {
 	} ),
 };
 
+export const optionsEntityMap = {
+	default: {
+		value: 'DTT_ID',
+		label: ( entity ) => {
+			return prettyDateFromDateTime( entity );
+		},
+	},
+	name: {
+		value: 'DTT_ID',
+		label: 'DTT_name',
+	},
+};
+
```

Comment:
As discussed in slack, I'd like to see all these `optionsEntityMap` moved from the current `model/{modelName}/query.js` location, into the `compnents/form/select/model-selects/{modelName}.js` file.